omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -101,7 +102,9 @@
     <SetProperty Id="PROJECTLOCATIONONCMDLINESLASH" Value="[PROJECTLOCATION]\" Before="AppSearch">PROJECTLOCATION </SetProperty>
     <SetProperty Id="CONFDIRONCOMMANDLINE" Value="[APPLICATIONDATADIRECTORY]" Before="AppSearch">APPLICATIONDATADIRECTORY </SetProperty>
     <SetProperty Id="CONFDIRONCOMMANDLINESLASH" Value="[APPLICATIONDATADIRECTORY]\" Before="AppSearch">APPLICATIONDATADIRECTORY </SetProperty>
-
+    
+    <SetProperty Action="SetNPMOff" Id="NPMFEATURESTATE" Before="SetFinalizeProperty" Value="off" Sequence="execute"><![CDATA[not (&NPM = 3)]]></SetProperty>
+    <SetProperty Action="SetNPMOn"  Id="NPMFEATURESTATE" Before="SetFinalizeProperty" Value="on"  Sequence="execute"><![CDATA[(&NPM = 3)]]></SetProperty>
```

Comment:
I wonder if specifying NPM values or perhaps their location would be useful here. Strange to see a comparison to 3 IMHO. Just a thought.

Reply:
The "3" is a magic number provided by wix
https://www.firegiant.com/wix/tutorial/com-expression-syntax-miscellanea/expression-syntax/
(skip down to "prefixes")

Reply:
Oh boy, thank you Derek. My comments shows WiX ignorance :)

Reply:
However, your comment/question indicates I was remiss in describing what that meant.  Added a very verbose comment.